backend/src/graph-info/graph-info.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Controller, Get } from '@nestjs/common';
+import { GraphInfoService } from './graph-info.service';
+import { NodeType } from '../shared/graph-information/NodeType';
+import { EdgeType } from '../shared/graph-information/EdgeType';
+
+@Controller('graphInfo')
+export class GraphInfoController {
+  constructor(private readonly graphInfoService: GraphInfoService) {}
+
+  /**
+   * Returns information about all edges of a graph
```

Comment:
Maybe rather smth like "gets information about all edges from GraphInfoService" since this method only calls the Service. So we have a better distinction in comments between controller and service.

Reply:
Implementation details should be hidden.
For a caller it's only relevant what a method does, not how. I Think its find like that

Reply:
okay I get your point you are probably right